silhouette-password-bcrypt/src/test/scala/com/mohiva/play/silhouette/password/BCryptPasswordHasherSpec.scala

```diff
@@ -17,56 +17,82 @@ package com.mohiva.play.silhouette.password
 
 import com.mohiva.play.silhouette.api.util.PasswordInfo
 import org.specs2.mutable.Specification
+import org.specs2.specification.Scope
 
 /**
  * Test case for the [[BCryptPasswordHasher]] class.
  */
 class BCryptPasswordHasherSpec extends Specification {
 
   "The `hash` method" should {
-    "hash a password" in {
-      val password = "my_S3cr3t_p@sswQrd"
-      val hasher = new BCryptPasswordHasher
-      val info = hasher.hash(password)
-
-      info must beAnInstanceOf[PasswordInfo]
-      info.hasher must be equalTo BCryptPasswordHasher.ID
-      info.password must not be equalTo(password)
-      info.salt must beNone
+    "hash a password" in new Context {
+      override val passwordInfo = hasher.hash(password)
+
+      passwordInfo must beAnInstanceOf[PasswordInfo]
+      passwordInfo.hasher must be equalTo BCryptPasswordHasher.ID
+      passwordInfo.password must not be equalTo(password)
+      passwordInfo.salt must beNone
     }
   }
 
   "The `matches` method" should {
-    "return true if a password matches a previous hashed password" in {
-      val password = "my_S3cr3t_p@sswQrd"
-      val hasher = new BCryptPasswordHasher
-      val info = hasher.hash(password)
+    "return true if a password matches a previous hashed password" in new Context {
+      hasher.matches(passwordInfo, password) must beTrue
+    }
+
+    "return false if a password doesn't match a previous hashed password" in new Context {
+      hasher.matches(passwordInfo, "not-equal") must beFalse
+    }
+  }
 
-      hasher.matches(info, password) must beTrue
```

Comment:
If the test runs in the CI system it's not. I think the test result should so meaningful that no further debugging, which values has failed, is necessary.

Or do you mean, add the `println` to the test in general?

```scala
"test" in new Scope {
  val list = Seq(true, true, false, false)

  for (v <- list) {
    println(v)
    v must beTrue
  }

}
```

Prints:
```
true
true
false
```

And not:
```
true
true
false
false
```

Because the test stops after the erroneous value.